salesforce-composite/2.7.0/modules/ROOT/pages/salesforce-composite-connector-reference.adoc

```diff
@@ -0,0 +1,1366 @@
+| Target Value a| String |  Expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Object
+|===
+
+=== For Configurations
+
+* <<composite-config>>
+
+==== Throws
+
+* SALESFORCE-COMPOSITE:CONNECTIVITY
+* SALESFORCE-COMPOSITE:INVALID_SESSION
+* SALESFORCE-COMPOSITE:RETRY_EXHAUSTED
+* SALESFORCE-COMPOSITE:TRANSACTION
+* SALESFORCE-COMPOSITE:TRANSFORMATION
+* SALESFORCE-COMPOSITE:UNKNOWN
+* SALESFORCE-COMPOSITE:VALIDATION
+
+[[postGetLimits]]
+=== Post Get Limits
+`<salesforce-composite:post-get-limits>`
+
+Provide metadata for the result of a create. This operation takes as input one record from the result returned by `executeCompositeBatch(String, java.util.List)`, which belongs to a create subrequest generated by `preCreate(String, java.util.Map)`.
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Get Limits Result a| Object |  The result of the create subrequest. |  `#[payload]` |
+| Target Variable a| String |  Name of the variable in which to store the operation's output |  |
+| Target Value a| String |  Expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Object
+|===
+
+=== For Configurations
+
+* <<composite-config>>
+
+==== Throws
+
+* SALESFORCE-COMPOSITE:CONNECTIVITY
+* SALESFORCE-COMPOSITE:INVALID_SESSION
+* SALESFORCE-COMPOSITE:RETRY_EXHAUSTED
+* SALESFORCE-COMPOSITE:TRANSACTION
+* SALESFORCE-COMPOSITE:TRANSFORMATION
+* SALESFORCE-COMPOSITE:UNKNOWN
+* SALESFORCE-COMPOSITE:VALIDATION
+
+[[postQuery]]
+=== Post Query
+`<salesforce-composite:post-query>`
+
+Provide metadata for the result of a query. This operation takes as input one record from the result returned by `executeCompositeBatch(String, java.util.List)`, which belongs to a query subrequest generated by `preQuery(String, java.util.Map)`.
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Query a| String |  The query that was used in the `preQuery` operation - needed for metadata. |  | x
+| Query Result a| Object |  The result of the query subrequest. |  `#[payload]` |
+| Streaming Strategy a| * <<repeatable-in-memory-iterable>>
+* <<repeatable-file-store-iterable>>
+* non-repeatable-iterable |  Configure to use repeatable streams. |  |
```

Comment:
From the spreadsheet: Configure how Mule processes streams with streaming strategies. Repeatable streams are the default behavior.